apps/api/src/form/entity/form.entity.ts

```diff
@@ -5,15 +5,16 @@ import { FormPayloadDTO } from '@ehpr/common';
 
 @Entity('form')
 export class FormEntity extends BaseEntity {
+  @Exclude()
   @Column('jsonb', { nullable: false })
   payload!: FormPayloadDTO;
 
   @Column('varchar', { nullable: true })
   confirmationId!: string;
 
   @Exclude()
-  @Column('timestamp', { nullable: true })
-  notifiedAt?: Date;
+  @Column('varchar', { nullable: true })
+  chesId?: string;
```

Comment:
Determined we don't need a `notifiedAt` column? I don't think it was a requirement right>

Reply:
would `chesTransactionId` be better?

Reply:
I figured saving timestamp is pointless as both actions happen at the same time. And yes `chesTransactionId` would be better